lib/ansible/modules/messaging/rabbitmq_policy.py

```diff
@@ -51,13 +51,15 @@
     version_added: "2.1"
   pattern:
     description:
-      - A regex of queues to apply the policy to.
-    required: true
+      - A regex of queues to apply the policy to. Required when
+        C(state=present).
+    required: false
```

Comment:
@bcoca I have updated description. Is there 'version_changed' tag like 'version_added'?